release/releaser.go

```diff
@@ -307,20 +281,38 @@ func filters(spec *flux.ReleaseSpec, conf instance.Config) ([]ServiceFilter, err
 		imgFilt := &SpecificImageFilter{id}
 		filtList = append(filtList, imgFilt)
 	}
-	if spec.ServiceSpec != flux.ServiceSpecAll {
-		svcID, err := flux.ParseServiceID(string(spec.ServiceSpec))
+
+	// Service filter
+	ids := []flux.ServiceID{}
+	for _, s := range spec.ServiceSpecs {
+		if s == flux.ServiceSpecAll {
+			break
+		}
+		id, err := flux.ParseServiceID(string(s))
 		if err != nil {
 			return nil, err
 		}
-		incFilt := &IncludeFilter{[]flux.ServiceID{svcID}}
+		ids = append(ids, id)
+	}
+	if len(ids) > 0 {
+		incFilt := &IncludeFilter{ids}
```

Comment:
afaict, in the instance of: `spec.ServiceSpecs = []string{"some-id", "<all>"}`, this will scope to just `some-id`, instead of letting `<all>` override that (like currently).
Not sure it is a bug, or if we should fix it, just checking that is what you meant.